nixos/modules/services/x11/desktop-managers/gnome.md

```diff
@@ -116,6 +116,37 @@ Unfortunately, we lack a way for these to be managed in a completely declarative
 So you have to enable them manually with an Extensions application.
 It is possible to use a [GSettings override](#sec-gnome-gsettings-overrides) for this on `org.gnome.shell.enabled-extensions`, but that will only influence the default value.
 
+## Nautilus Extensions {#sec-gnome-nautilus-extensions}
+
+Nautilus extensions can add additional functionality to Nautilus, usually via the right-click menu. Some of these extensions are dynamically loaded from `$NAUTILUS_4_EXTENSIONS_DIR` environment variable, which is automatically set if [](#opt-services.gnome.core-utilities.enable) is true (enabled by default if gnome is enabled).
```

Comment:
The environment variable is an implementation detail so I am not sure we should expose it. Ideally, we would drop it in favour of Nautilus wrapper package.

Reply:
I wonder if we should have a Nautilus module in the meanwhile.